syntax = "proto3";

package render_rob_cache;

message RenderRobCache {
    string current_file = 1;
    repeated string recent_files = 2;
}